snippets/arm-templates/deployments/with-userassigned-identity/README.md

```diff
@@ -18,7 +18,7 @@ This template provides a way to deploy a **NGINX Deployment** in a **Resource Gr
 If you're new to **NGINXaaS for Azure**, see:
 
 - [NGINXaaS for Azure Documentation](https://docs.nginx.com/nginxaas/azure/)
-- [NGINXaaS for Azure Managed Identity Documentation](https://docs.nginx.com/nginxaas/azure/management/managed-identity/)
+- [NGINXaaS for Azure Managed Identity Documentation](https://docs.nginx.com/nginxaas/azure/getting-started/managed-identity/)
 
 If you're new to **Managed Identities**, see:
 
```

Comment:
Why does the link on 21 for Azure Managed Identity Documentation take me to a different location (link)  than the section starting on line 23?

Reply:
A bit confused. They are supposed to be different links. One is for the NGINXaaS created documentation and the other is for the Azure created documentation. Do you find the naming of the links to be confusing?